### PR TITLE
Replace try / catch blocks in tests with the QUnit throws() assertion.

### PR DESCRIPTION
**Pull request by ronyeh**
Fixes https://github.com/0xfe/vexflow/issues/1130

### CODE REVIEW

**Comment by ronyeh** [comment]
OK, hopefully this fixes the build issue! I added a single space in the RuntimeError message string to make it easier to read.

I converted all the try / catch blocks to QUnit's throws().

For future reference, the official recommendation for try / catch blocks is to set the type of `e` to `unknown` and then do a type check before using it:

https://www.typescriptlang.org/play?ts=4.0.2#example/unknown-in-catch

```
try {
  // ..
} catch (e: unknown) {
  // You cannot use `e` at all until the type system learns what it is.
  e.stack; // <= compiler will complain!

  if (e instanceof SyntaxError) {
    e.stack;
  }
}
```

File: tests/keysignature_tests.ts [hunk 1]
```diff
@@ -52,7 +48,6 @@ function parser(): void {
   catchError('b');
   catchError('Kb');
   catchError('Fb');
-  catchError('Ab');
```
Comment (rvilarl) [inline tests/keysignature_tests.ts:55]: Is this removal intentional?

**Review by rvilarl — CHANGES_REQUESTED** [review]

File: tests/keysignature_tests.ts (hunk 1 above)
Comment (ronyeh) [reply tests/keysignature_tests.ts:55]: Thanks for the review! Yes, it is intentional.

The previous catchError() method would do nothing if there was no exception thrown. When I changed it to QUnit's `throws()`, it is now asserting that an exception WILL be thrown. If the line of code succeeds, the assertion fails.

`Flow.keySignature('Ab')` is a valid key signature. It also appears 7 lines down, and runs without throwing any exception.

**Comment by 0xfe** [comment]
Thanks for the quick fix. The code is also easier to read.
